syft/core/hooks/torch/hook.py

```diff
@@ -585,28 +462,17 @@ def _hook_tensor(self, tensor_type):
                     setattr(tensor_type, attr, new_attr)
 
         # Add in our own Grid-specific methods
-        self._hook_tensor_send_(tensor_type)
+        self._hook_send_(tensor_type)
         self._hook_get_(tensor_type)
         self._hook_tensor__ser(tensor_type)
 
-    def _hook_tensor___init__(hook_self, tensor_type):
```

Comment:
Why is this no longer here?  If we're not using it, how are we registering newly created tensors?

Reply:
Figured out how to call the same function for tensor and var.